onnx/backend/test/case/node/averagepool.py

```diff
@@ -192,6 +222,37 @@ def export_averagepool_2d_same_upper():
 
         expect(node, inputs=[x], outputs=[y], name='test_averagepool_2d_same_upper')
 
+    @staticmethod
+    def export_averagepool_2d_same_upper_count_include_pad():
+        """
+        iutput_shape: [1, 3, 32, 32]
+        output_shape: [1, 3, 32, 32]
+        pad_shape: [1, 1] -> [0, 1, 0, 1] by axis
+        """
+        node = onnx.helper.make_node(
+            'AveragePool',
+            inputs=['x'],
+            outputs=['y'],
+            kernel_shape=[2, 2],
+            auto_pad='SAME_UPPER',
```

Comment:
seeing that autopad is now deprecated? why is this test being added?

Reply:
@anirudhacharya 
Tensorflow uses something like `auto_pad` and I am working on backend of tf. 
At least for test case coverage for tf, I added this. 
Of course I am ok with removing it.

Reply:
It would be good to remove this and then evaluate a plan when the deprecated attributes/operators will be EOL'ed.